src/bitops.c

```diff
@@ -1200,7 +1198,7 @@ void bitfieldGeneric(client *c, int flags) {
                 int64_t oldval, newval, wrapped, retval;
                 int overflow;
 
-                oldval = getSignedBitfield(o->ptr,thisop->offset,
+                oldval = getSignedBitfield(o->ptr, thisop->offset,
```

Comment:
we should avoid formatting unless we modify it.

```suggestion
                oldval = getSignedBitfield(o->ptr,thisop->offset,
```